src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,58 @@
+
+fun main(args:Array<String>){
+
+    var dataIO = DataIO()
```

Comment:
Die Methode macht genau das, was `external` machen soll, d.h. Daten aus CSV lesen, simulieren und wieder rein schreiben. Sehr gut. Verschieb den Code einfach in eine neue Methode `external` und der Teil ist erledigt.

Fehlt noch `scenario`, prinzipiell das gleiche aber ohne CSV machen soll, d.h. eine Simulation mit willkürlich gewählten Daten erstellen, simulieren und Ergebnisse auf die Konsole schreiben.